portal/src/graphql/adminapi/AuditLogScreen.tsx

```diff
@@ -369,10 +369,10 @@ const AuditLogScreen: React.VFC = function AuditLogScreen() {
   }, []);
 
   const userNodeIDs = useMemo(() => {
-    return debounedSearchUserID
-      ? [toTypedID(NodeType.User, debounedSearchUserID)]
+    return debounedSearchQuery
+      ? [toTypedID(NodeType.User, debounedSearchQuery)]
       : null;
-  }, [debounedSearchUserID]);
+  }, [debounedSearchQuery]);
```

Comment:
This does not seem correct. Now that we call it search query, calling toTypedID on a search query does not make much sense.

Suggestion: You check whether the input looks like an email address or a phone number. If it looks like a email address, then you call the API with `emailAdresses: [emailAdress]`. If it looks like a phone number, then you call the API with `phoneNumbers: [phoneNumber]`. Otherwise, we assume it is a user ID, and call the API with `userIDs: [nodeUserID]`.

Reply:
```ts
export function validatePhoneNumber(inp: string): string | null {
  const lookLike = /^ *\+[0-9 -]*$/.test(inp);
  if (lookLike) {
    return inp.trim();
  }
  return null;
}
```

Reply:
^\s*\S+@\S+\s*$